### PR TITLE
add kshitijarvikar

### LINKED ISSUE #833: Kshitij Arvikar
github

### PR DESCRIPTION
**Pull request by kshitijarvikar**
resolve #833

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -1570,5 +1570,11 @@
     "githubUsername": "gouravkhunger",
     "imageUrl": "https://user-images.githubusercontent.com/46792249/141504841-54ecc6ab-437f-4122-bdef-2127e264bf7e.png",
     "issueNumber": 831
-  }
+  },
+  {
+    "name": "Kshitij Arvikar",
+    "githubUsername": "kshitijarvikar",
+    "imageUrl": "![github](https://user-images.githubusercontent.com/89210111/141609753-c7666dfe-91df-4576-8d6a-495b475e3ae7.png)",
```
Comment (schmelto) [inline data.json:1577]:
```suggestion
    "imageUrl": "https://user-images.githubusercontent.com/89210111/141609753-c7666dfe-91df-4576-8d6a-495b475e3ae7.png",
```

File: data.json [hunk 2]
```diff
@@ -1570,5 +1570,11 @@
...
+    "issueNumber": 833
+  }
```
Comment (schmelto) [inline data.json:1579]:
```suggestion
  }
```

**Review by schmelto — CHANGES_REQUESTED** [review]

**Comment by kshitijarvikar** [comment]
I have updated the requested changes. I hope its good now.

File: data.json [hunk 3]
```diff
@@ -1570,5 +1570,11 @@
...
     "imageUrl": "https://user-images.githubusercontent.com/46792249/141504841-54ecc6ab-437f-4122-bdef-2127e264bf7e.png",
     "issueNumber": 831
-  }
+  },
+  {
+    "name": "Kshitij Arvikar",
+    "githubUsername": "kshitijarvikar",
+    "imageUrl": "https://user-images.githubusercontent.com/89210111/141609753-c7666dfe-91df-4576-8d6a-495b475e3ae7.png",
+    "issueNumber": 833
+  }
```
Comment (schmelto) [inline data.json:1579]:
```suggestion
  }
```

**Review by schmelto — APPROVED** [review]
Lgtm

**Comment by schmelto** [comment]
I made one small little change

**Comment by schmelto** [comment]
**Thanks for your contribution** 🚀

**Comment by kshitijarvikar** [comment]
> I made one small little change 

Thank you so much for guiding me. It was my first PR 😃

**Pull request merged**